packages/color-slider/test/color-slider.test.ts

```diff
@@ -229,7 +229,6 @@ describe('ColorSlider', () => {
 
         await elementUpdated(el);
 
-
```

Comment:
I don't know why some files were submitted together with my PR, they seem to be changes after linting...